app/jobs/reports/sp_user_counts_report.rb

```diff
@@ -15,7 +15,37 @@ def perform(_date)
       user_counts = transaction_with_timeout do
         Db::Identity::SpUserCounts.call
       end
+
+      track_report_data_events(user_counts)
       save_report(REPORT_NAME, user_counts.to_json, extension: 'json')
     end
+
+    private
+
+    def track_report_data_events(user_counts)
+      total_ial1_count = 0
+      total_ial2_count = 0
+      user_counts.each do |hash|
+        ial1_count = hash['ial1_total'].to_i
+        ial2_count = hash['ial2_total'].to_i
+        track_report_data_event(
+          Analytics::REPORT_SP_USER_COUNTS,
+          issuer: hash['issuer'],
+          user_total: hash['total'].to_i,
+          ial1_user_total: ial1_count,
+          ial2_user_total: ial2_count,
+          app_id: hash['app_id'].to_s,
+        )
+        total_ial1_count += ial1_count
+        total_ial2_count += ial2_count
+      end
+
+      track_report_data_event(
+        Analytics::REPORT_TOTAL_SP_USER_COUNTS,
+        user_total: total_ial1_count + total_ial2_count,
```

Comment:
I don't think it makes sense to blindly add ial1 and ial2 users together... if we had the UUIDs we could do a unique count here but just adding the two doesn't make sense. Plus, in any Cloudwatch query we could easily add the two together if we wanted